config/routes.rb

```diff
@@ -0,0 +1,15 @@
+Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+  root 'root#index'
+
+  namespace :api do
+    namespace :v1 do
+      resources :doctors, only: [:index, :show, :create, :update]
+      resources :users
+      resources :appointments
```

Comment:
- [ ] OPTIONAL: Kindly, ensure you are not generating more routes than those that are needed.


Kindly check if `resources :users` is needed.  You are not using any of index, show, edit, create, update and destroy methods in your users controller.

Kindly check if 'resources :appointments' is needed without an `only:` argument.  


That way, your routes table will be simpler, and your app will be faster. 👍